CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 - `postgresqlreceiver`: Fix issue where empty metrics could be returned after failed connection (#7502)
 - `resourcetotelemetry`: Ensure resource attributes are added to summary
   and exponential histogram data points. (#7523)
+- `tailsamplingprocessor`: And policy only works as a sub policy under a composite policy (#7590)
```

Comment:
Given that this bug hasn't been part of any versions, as the policy has just been added, it's safe to leave this out.